cpp/dolfin/fem/utils.cpp

```diff
@@ -121,9 +121,9 @@ la::PETScMatrix dolfin::fem::create_matrix(const Form& a)
   }
 
   // Get dof maps
-  std::array<const DofMap*, 2> dofmaps
-      = {{a.function_space(0)->dofmap().get(),
-          a.function_space(1)->dofmap().get()}};
+  std::array<std::shared_ptr<const DofMap>, 2> dofmaps
```

Comment:
I changed from pointer to shared_ptr as there was no need to getting raw pointers anymore.